bundlearchive_test.go

```diff
@@ -36,6 +36,18 @@ func (s *BundleArchiveSuite) TestReadBundleArchiveBytes(c *gc.C) {
 
 	archive, err := charm.ReadBundleArchiveBytes(data)
 	c.Assert(err, gc.IsNil)
+	c.Assert(archive.ContainsOverlays(), gc.Equals, false)
+	checkWordpressBundle(c, archive, "")
+}
+
+func (s *BundleArchiveSuite) TestReadMultiDocBundleArchiveBytes(c *gc.C) {
+	path := archivePath(c, readBundleDir(c, "wordpress-simple-multidoc"))
+	data, err := ioutil.ReadFile(path)
+	c.Assert(err, gc.IsNil)
+
+	archive, err := charm.ReadBundleArchiveBytes(data)
+	c.Assert(err, gc.IsNil)
+	c.Assert(archive.ContainsOverlays(), gc.Equals, true)
```

Comment:
There is "jc.IsTrue" and "jc.IsFalse" which gives slightly better output when it doesn't match (though maybe we don't depend on github.com/juju/testing/checkers here)

Reply:
I guess it's just muscle memory :D 

bundle_test.go does import jc so since we already depend on checkers I will update my code to use that instead.

Reply:
Pushed https://github.com/juju/charm/pull/287/commits/1d63110874911321b8ac69154e247fee7c728d6f to address this comment.